src/lib/features/authenticate/ui/specs/sign-out-button.spec.ts

```diff
@@ -0,0 +1,16 @@
+import SignOutButton from '../sign-out-button.svelte';
+import { render } from '@testing-library/svelte';
+import userEvent from '@testing-library/user-event';
+
+jest.mock('../../model/sign-out', () => ({
+  signOut: jest.fn(),
+}));
+
+import { signOut } from '../../model/sign-out';
+
+it('renders and clicks the sign out button', async () => {
+  const user = userEvent.setup();
+  const { getByRole } = render(SignOutButton);
+  await user.click(getByRole('button', { name: 'Sign out' }));
+  expect(signOut).toHaveBeenCalledTimes(1);
```

Comment:
nitpick (non-blocking): I think the number of times this function has been called is not that important in this case, there's a shorter `.toHaveBeenCalled()` assertion. But this works too, and asserts even more things